apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -293,4 +293,9 @@ func (r *OpenStackControlPlane) DefaultServices() {
 	if r.Spec.Placement.Enabled {
 		r.Spec.Placement.Template.Default()
 	}
```

Comment:
To handle missing deps for heat can also update ValidateServices and checkDepsEnabled above.